baybe/utils/dataframe.py

```diff
@@ -413,3 +414,28 @@ def fuzzy_row_match(
             inds_matched.extend(inds_found)
 
     return pd.Index(inds_matched)
+
+
+def pretty_print_df(df: pd.DataFrame) -> str:
+    """Return a pretty/readable str object.
+
+    This function returns the first 10 rwos and 7 columns from the dataframe.
+    In case the dataframe is empty, it returns a corresponding statement.
+
+    Args:
+        df: The dataframe to be printed.
+
+    Returns:
+        The values to be printed as a str table.
+    """
+    # Handling with empty dataframe
+    if len(df) == 0:
+        return "This attribute is empty."
+
+    # Get first 10 rows and 7 columns and do not break the table in two lines
+    with option_context(
+        "display.max_rows", 10, "display.max_columns", 7, "expand_frame_repr", False
+    ):
+        str_df = f"{df}"
+        pass
```

Comment:
Is this `pass` here necessary? You should be able to just remove it, this should not make a difference.

Reply:
No, `pass` is not necessary. Just I removed it 👍